app/src/main/java/com/github/se/studybuddies/data/todo/ToDo.kt

```diff
@@ -0,0 +1,26 @@
+package com.github.se.studybuddies.data.todo
+
+import com.github.se.studybuddies.data.Location
+import java.time.LocalDate
+
+data class ToDo(
+    val uid: String,
+    val name: String,
+    val assigneeName: String,
+    val dueDate: LocalDate,
+    val location: Location,
+    val description: String,
+    val status: ToDoStatus

```

Comment:
I removed location and assigneeName, I wanted to see if maybe the feature was of any interest since it was already implemented in the bootcamp and not difficult to reproduce
For the status one, I am not removing it yet because it might come useful, we should discuss this tomorrow